project.clj

```diff
@@ -3,8 +3,7 @@
   :url "http://example.com/FIXME"
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[kixi/gg4clj "0.1.1-SNAPSHOT" :exclusion [org.clojure/clojure]]
-                 [org.clojure/clojure "1.8.0"]
+  :dependencies [[org.clojure/clojure "1.8.0"]
```

Comment:
What did this do?

Reply:
I just noticed the dependency was duplicated, so I took one of the instances out as housekeeping.

Reply:
ah ok sorry, wasn't clear until I looked at the whole file